src/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/SphereTransition.hpp

```diff
@@ -29,7 +29,7 @@ namespace domain::CoordinateMaps::ShapeMapTransitionFunctions {
 class SphereTransition final : public ShapeMapTransitionFunction {
  public:
   explicit SphereTransition() = default;
-  SphereTransition(double r_min, double r_max);
+  SphereTransition(double r_min, double r_max, bool reverse = false);
```

Comment:
Add docs above about what reverse does and the new coefficient values for $a$ and $b$